src/mote/templates/e404page.html

```diff
@@ -36,21 +36,48 @@
                 </div>
             </div>
         </nav>
+        <div class="container-lg" style="padding-top: 20px;">
+            <div class="card mt-2 mb-2">
+                <div class="card-body text-center">
+                    <img src="{{ url_for('static', filename='imgs/404_not_found.svg') }}" style="width: 40%; height: 40%;" />
```

Comment:
It works fine on Chrome, but have some extra padding at the bottom on Firefox.
One way to fix that is by removing the height css attribute on the 404 image (which is not really needed anyway):
```suggestion
                    <img src="{{ url_for('static', filename='imgs/404_not_found.svg') }}" style="width: 40%;" />
```